flowetl/etl/tests/test_render_sql_callable.py

```diff
@@ -0,0 +1,55 @@
+from unittest.mock import patch, MagicMock
+from pathlib import Path
+
+from airflow import DAG
+from airflow.operators import BaseOperator
+
+from etl.production_task_callables import render_sql_callable
+
+
+class MockDagRun:
+    def __init__(self, conf):
+        self.conf = conf
+
+
+def test_render_sql_callable(tmpdir):
+    """
+    Test that the render sql callable is able to construct the
+    correct sql from a template in the correct location and issues
+    this sql to the db_hook run command.
+    """
+
+    # Make temp sql template in "config/etl/voice/init.sql" where
+    # config is a temp dir
+
+    config_dir = tmpdir.mkdir("config")
+    etl_dir = config_dir.mkdir("etl")
+    voice_dir = etl_dir.mkdir("voice")
+
+    template_name = "init"
+    init_template_path = voice_dir.join(f"{template_name}.sql")
+    init_template = "select {{number}}"
+    init_template_path.write(init_template)
+
+    # Mocks so we can use the templating feature of the operator this
+    # callable runs in and a mock of the db_hook which will run the sql
+    mock_dag = DAG(dag_id="testing", default_args={"start_date": "2016-01-01"})
+    mock_task_op = BaseOperator(task_id="testing", dag=mock_dag)
+    mock_pghook = MagicMock()
+
+    # Mock of the config passed to the dag_run
+    conf = {"number": 23, "template_path": Path("etl/voice")}
+    mock_dag_run = MockDagRun(conf=conf)
+
+    render_sql_callable(
```

Comment:
Just to be sure I'd also check that no calls have happened before the the call to `render_sql_callable`:
```suggestion
    assert mock_pghook.mock_calls == []
   
    render_sql_callable(
```